modules/sagemaker/sagemaker-templates-service-catalog/README.md

```diff
@@ -49,9 +49,18 @@ The template contains an example CI/CD pipeline to deploy the model endpoints to
 
 The template is based on basic multi-account template from [AWS Enterprise MLOps Framework](https://github.com/aws-samples/aws-enterprise-mlops-framework/blob/main/mlops-multi-account-cdk/mlops-sm-project-template/README.md#sagemaker-project-stack).
 
+## Prerequesites:
+#TODO: Update detials for GitHub Access token and AWS CodeConnection
+- Target AWS account should contain AWS Secret Manager secret that contains GitHub personal access token with required permissions to manage repository. Refer guide [Creating a fine-grained personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) in order to create access token.
```

Comment:
Should we document what repositories might get created or any other information about them? Personally, I'd be terrified this is going to stomp on a repository that I previously created.

Reply:
Added information about how GitHub repository will be created in conjunction with SageMaker project name.